testutil/util.go

```diff
@@ -35,6 +35,10 @@ type T struct {
 	teardownActions []func()
 }
 
+func (t *T) FakeRun(command string) *FakeCmd {
```

Comment:
Is this used anywhere?

Reply:
oops, should have been. Let me push an update